src/ConfigSetup.cpp

```diff
@@ -1003,6 +1003,858 @@ void ConfigSetup::Init(const char *fileName)
   printf("%-40s %-s\n\n", "Finished Reading Input File:", fileName);
 }
 
+#if GOMC_LIB_MPI
+void ConfigSetup::Init(const char *fileName, MultiSim *& multisim)
+{
```

Comment:
@ynejahi @go2432 There should be a better way than duplicating the whole input reader. The ConfigSetup.cpp is extremely long and confusing. It would cause a problem If someone once to add a new parameter.

Reply:
void ConfigSetup::Init(const char fileName, MultiSim constconst& multisim)
I used the =NULL default in the Simulation constructor.
To index into a vector for temperature or chemPot, line size must be a certain size (>2, >3 respectively) && multisim != NULL